tslearn/metrics.py

```diff
@@ -362,10 +361,44 @@ def njit_accumulated_matrix_subsequence(subseq, longseq):
 
 
 @njit()
-def _return_path_subsequence(acc_cost_mat):
+def calculate_subsequence_path(acc_cost_mat, idx_path_end):
```

Comment:
`calculate_subsequence_path` is a bit of a complicated name for public API no? What about `subsequence_path`?

Reply:
I renamed them to `subsequence_path` and `subsequence_cost_matrix`